canonical/snapcraft.io
# Update site navigation

## Done
Updated the site navigation

## How to QA
- Go to https://snapcraft-io-4209.demos.haus/
- Check that the site navigation looks and works as expected
